src/features/MoviesList/useMoviesData.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react";
+
+const nowPlaying = "https://api.themoviedb.org/3/movie/now_playing?language=en-US&page=1&region=pl";
```

Comment:
I don't think it's being used anywhere.. 😅

Reply:
indeed... :D 

deleted !